src/platform/plugins/shared/presentation_util/public/components/labs/project_list_item.tsx

```diff
@@ -119,3 +120,61 @@ export const ProjectListItem = ({ project, onStatusChange }: Props) => {
     </EuiFlexItem>
   );
 };
+
+const useStyles = () => {
+  const { euiTheme } = useEuiTheme();
+  const styles = useMemo(() => {
+    return {
+      projectListItemStyles: css({
+        position: 'relative',
+        background: euiTheme.colors.emptyShade,
+        padding: euiTheme.size.l,
+        minWidth: '500px',
+        '+ .projectListItem:after': {
```

Comment:
Would a `& + &:after` selector work here (instead of `+ .projectListItem:after`)? Or will that not work with the way this Emotion is structured?